android/assets/jsons/translations/template.properties

```diff
@@ -625,7 +625,8 @@ Show minimap =
 off = 
 Show pixel units = 
 Show pixel improvements = 
-Enable nuclear weapons = 
+Enable Nuclear Weapons = 
+Demographics Scoreboard\n(replaces Rankings)" = 
```

Comment:
"Experimental Demographics scoreboard" - this should be for the testing phase only, after which it will be on by defualt

Reply:
Done.